scripts/apps/authoring/metadata/views/metadata-widget.html

```diff
@@ -37,7 +37,7 @@
             </li>
             <li class="terms-box" ng-if="!$root.config.features.alchemy">
                 <label translate>Keywords</label>
-                <div sd-meta-words-list data-item="item" data-field="keywords" data-change="autosave(item)" data-header="_editable" data-list="metadata.keywords" data-disabled="!_editable"></div>
```

Comment:
@ioanpocol we have list of predefined keywords 
https://github.com/superdesk/superdesk-aap/blob/master/server/data/vocabularies.json#L233

The `sd-meta-words-list` allows you to define arbitrary keywords as well as predefined one. Does the new one does the same?